doc/programmers/maintenance.rst

```diff
@@ -2,16 +2,23 @@ Maintenance
 ===========
 
 Description and how-to for maintenance operations.
-Some of the maintenance scripts have been moved to the `pcmsolvermeta
-repository <https://gitlab.com/PCMSolver/pcmsolvermeta>`_
```

Comment:
That `pcmsolvermeta` repo should actually be moved to GitHub. Can you do it? Or maybe open an issue.

Reply:
It seems like it is only a git repo, without issues etc. So I only have to push that repo to a new github repo? Then I can do it.

Reply:
Yep, it's only a repo. There are some pretty large files there though. I suggest you use [Git LFS](https://git-lfs.github.com/) so that GitHub doesn't complain or block the push.

Reply:
I do not have access to make a new repo on github/pcmsolver. Only you and luca.